ios/test_app.rb

```diff
@@ -38,6 +38,18 @@ def bundle_identifier(project_root, target_platform)
   @test_app_bundle_identifier
 end
 
+def react_native_path_from_manifest(project_root, target_platform) 
```

Comment:
Can we add tests for this?

Reply:
For adding positive cases for `react_native_path_from_manifest` will require changes in manifest, Do we use any mock manifest while running the test?

Reply:
Yes, look under `/test/__fixtures__`

Reply:
Alright, got it. I will add the tests there.

Reply:
Added test cases.